src/app/messages/en/messages.json

```diff
@@ -1,5 +1,7 @@
 {
   "i18n:a11y:skipToContent": "Skip to Content",
+  "i18n:a11y:accessibilityHelp": "Go to accessibility help page",
```

Comment:
If ["Go to accessibility help page" is right](https://github.com/openstax/unified/issues/1496#issuecomment-785086424), then it looks good to me.

Reply:
updated to "Go to accessibility page"